doc/manual/developing_a_ndsp.rst

```diff
@@ -169,6 +169,26 @@ The program below exemplifies how to use logging: ::
         main()
 
 
+Integration with artiq experiments
+----------------------------------
+
+To integrate the controller into an experiment, simply add an entry into the ``device_db.py`` following the example shown below for the ``aqctl_hello.py`` from above: ::
+
+	device_db.update({
+    	"hello": {
+        	"type": "controller",
+        	"host": "::1",
+        	"port": 3249,
+        	"command": "python aqctl_hello.py -p {port}" 
```

Comment:
Absolute paths or a python package with the controller would be better. This device_db entry may or may not work depending on the working directory where the ``artiq_ctlmgr`` command was started (note that it can be started e.g. with systemd).